plugin/core/context.py

```
@@ -92,4 +192,318 @@ def push_message(
+        # 抛出超时异常
+        raise TimeoutError(
+            f"Plugin {target_plugin_id} event {event_type}.{event_id} timed out after {timeout}s"
+        )
+
+    def query_plugins(self, filters: Optional[Dict[str, Any]] = None, timeout: float = 5.0) -> Dict[str, Any]:
+        self._enforce_sync_call_policy("query_plugins")
+        if self._plugin_comm_queue is None:
+            raise RuntimeError(
+                f"Plugin communication queue not available for plugin {self.plugin_id}. "
+                "This method can only be called from within a plugin process."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)